src/node_sockaddr.h

```diff
@@ -72,18 +72,50 @@ class SocketAddress {
       uv_tcp_t* handle,
       SocketAddress* addr = nullptr);
```

Comment:
The four overloads taking a non-const pointer are no longer necessary, right?

Reply:
Yeah, I'll be pulling those back out. This is still very much a WIP